install-deps.sh

```diff
@@ -516,6 +526,9 @@ EOF
         echo "Using zypper to install dependencies"
         zypp_install="zypper --gpg-auto-import-keys --non-interactive install --no-recommends"
         $SUDO $zypp_install systemd-rpm-macros rpm-build || exit 1
+        if [ "$INSTALL_EXTRA_PACKAGES" ]; then
+            $SUDO dnf install -y $INSTALL_EXTRA_PACKAGES
```

Comment:
not sure if `dnf` works for SuSE distros. probably should use `$SUDO $zypp_install` here?

Reply:
Good catch, will fix.

Reply:
Fixed.